### DISCUSSION TITLE
What cron formats are supported? and does it support timezones?

### DISCUSSION THREAD

**Discussion by viperfx** [quirrel-dev/quirrel#169] [Q&A] [answered]
Just curious what the cron format is supported in the backend, and where can we go to find more about what types of scheduling is supported. E.g. 1st of the month, Mondays at 10am PST etc...

and how does it handle timezones? Are all crons UTC? or can we specify a timezone in the CRON format too?

**Comment by Skn0tt** [comment, marked as answer]
Hi @viperfx!
- https://github.com/harrisiirak/cron-parser is used, refer to their docs for supported syntax. (TLDR: the same as https://crontab.guru, including @daily, @hourly etc.)
- [Crons are interpreted in UTC](https://docs.quirrel.dev/api/queue#cron-schedule).
- You can't specify a timezone in your Cron schedule.

**Reply by viperfx** [reply]
Ah I see. Okay well, that helps. 

Think there are some limitations though

Such as 
- is it possible to do 1st of the month at 10am? 

Thanks!

**Reply by Skn0tt** [reply]
Sure! 1st of month 10am: https://crontab.guru/#0_10_1_*_*

**Reply by viperfx** [reply]
Ah I can't believe i missed that! Thanks.

Will report back on my redwoodjs usage :) Just starting to use all the features of Quirrel
